File: nodejs/02-connect.js
'use strict';
const mysqlx = require('@mysql/xdevapi');
(async function() {
  try {
    const session = await mysqlx.getSession( {user:'root', password:'welcome1', host:'127.0.0.1', port:33100});
    console.log(session.inspect());
    process.exit();
  } catch (err) {
    console.log(err.message);
    process.exit(1);
  }
})();

File: python/solution/script-python-mysql/01-pythonPart-I/02-connect.py
from mysql.connector import (connection)
import sys

_pass='';
if len(sys.argv) > 1 :
	_pass=sys.argv[1]

cnx = connection.MySQLConnection(user='root', password=_pass, host='127.0.0.1', port=3306, database='mysql')

cnx.close()

